src/test/enhetstester/kotlin/no/nav/familie/ba/sak/kjerne/autovedtak/svalbardtillegg/AutovedtakSvalbardtilleggBegrunnelseUtilTest.kt

```diff
@@ -0,0 +1,355 @@
+                        person = barn,
+                        behandling = behandling,
+                        beløp = 500,
+                        sats = 500,
+                        ytelseType = YtelseType.SVALBARDTILLEGG,
+                    ),
+                )
+
+            val innvilgedeOgReduserteSvalbardtilleggPerioder =
+                finnInnvilgedeOgReduserteSvalbardtilleggPerioder(
+                    forrigeAndeler = emptyList(),
+                    nåværendeAndeler = nåværendeAndeler,
+                )
+
+            val innvilgedePerioder = innvilgedeOgReduserteSvalbardtilleggPerioder.first
+            val redusertePerioder = innvilgedeOgReduserteSvalbardtilleggPerioder.second
+
+            assertThat(redusertePerioder).isEmpty()
+            assertThat(innvilgedePerioder).hasSize(1)
+            assertThat(innvilgedePerioder.single()).isEqualTo(YearMonth.of(2025, 11))
+        }
+
+        @Test
+        fun `Skal differensiere mellom barn og returnere alle innvilgede perioder`() {
+            val barn = lagPerson(type = PersonType.BARN)
+            val barn2 = lagPerson(type = PersonType.BARN)
+            val behandling = lagBehandling()
+
+            val nåværendeAndeler =
+                listOf(
+                    lagAndelTilkjentYtelse(
+                        fom = YearMonth.of(2025, 11),
+                        tom = YearMonth.of(2025, 12),
+                        person = barn,
+                        behandling = behandling,
+                        beløp = 500,
+                        sats = 500,
+                        ytelseType = YtelseType.SVALBARDTILLEGG,
+                    ),
+                    lagAndelTilkjentYtelse(
+                        fom = YearMonth.of(2025, 10),
+                        tom = YearMonth.of(2025, 12),
+                        person = barn2,
+                        behandling = behandling,
+                        beløp = 500,
+                        sats = 500,
+                        ytelseType = YtelseType.SVALBARDTILLEGG,
+                    ),
+                )
+
+            val innvilgedeOgReduserteSvalbardtilleggPerioder =
+                finnInnvilgedeOgReduserteSvalbardtilleggPerioder(
+                    forrigeAndeler = emptyList(),
+                    nåværendeAndeler = nåværendeAndeler,
+                )
+
+            val innvilgedePerioder = innvilgedeOgReduserteSvalbardtilleggPerioder.first
+            val redusertePerioder = innvilgedeOgReduserteSvalbardtilleggPerioder.second
+
+            assertThat(redusertePerioder).isEmpty()
+            assertThat(innvilgedePerioder).hasSize(2)
+            assertThat(innvilgedePerioder).anySatisfy { assertThat(it).isEqualTo(YearMonth.of(2025, 10)) }
+            assertThat(innvilgedePerioder).anySatisfy { assertThat(it).isEqualTo(YearMonth.of(2025, 11)) }
+        }
+
+        @Test
+        fun `Ved flere barn som har lik innvilgeses tidspunkt returneres bare 1 periode`() {
```

Comment:
Nitpick, men endre det gjerne i filen for finnmarkstilleggbehandlinger også
```suggestion
        fun `Ved flere barn som har likt innvilgelsestidspunkt returneres bare 1 periode`() {
```